### DISCUSSION TITLE
Dictionary Download Site

### DISCUSSION THREAD

**Discussion by LucyOne** [koreader/koreader#7645] [Q&A] [answered]
Where does KOReader get its dictionaries from when you download them from inside the app?
I vaguely remember having found a download site on the web. I was able to download the dictionaries onto my PC because the downloads kept freezing when I tried them on my reading device.

**Comment by Frenzie** [comment, marked as answer]
KOReader contains a list of tailored *legal* dictionaries. Many came from https://tuxor1337.frama.io/firedict/dictionaries.html (see the wiki).

You can see the details for everything in https://github.com/koreader/koreader/blob/3d713dc34c0f568f1a028eaa951cd5c92da1176e/frontend/ui/data/dictionaries.lua
